### DISCUSSION TITLE
Any plans for adding documentation?

### DISCUSSION THREAD

**Discussion by lisenet** [frenck/home-assistant-config#97] [General]
You wrote "figure out how to document that all". Any plans for actually doing this? :)

**Comment by frenck** [comment]
Maybe, one day... Currently, I'm more occupied by creating stuff for and around the Home Assistant project itself. I feel like I can make a larger difference doing that, than working on my own home stuff.

**Reply by lisenet** [reply]
No worries, thank you for getting back to me. From my personal, and somewhat biased, work experience, no matter how good a project is or might be, its use and adoption largely depends on documentation. We all live busy lives and there isn't always time to learn things "the hard way". Good luck with the project!

**Reply by frenck** [reply]
You are looking at the configuration of my personal Home. This GitHub project has no other purpose than versioning my own Home Automation... for me and my family.

**Reply by lisenet** [reply]
Do you mind sharing the link to the "real" project?

**Reply by frenck** [reply]
Home Assistant?

https://www.home-assistant.io

**Reply by lisenet** [reply]
That's the one, thank you!
